SafeguardSessions/CONTRIBUTION.md

```diff
@@ -65,6 +65,13 @@ When writing **commit messages**, consider the following:
     - If it both has a work item in Azure DevOps and a GitHub issue: `References: azure #<azure-id>, #<github-id>`
 - For examples, view the [Commit history of the main branch]
 
+## Error codes
+
+The introduction of error codes is necessary to easily identify errors regardless of the language of the Power BI interface. When creating error codes, the following convention has been introduced:

```

Comment:
I can accept writing about error codes in the .md, but if we do, we should not write it in a language tone that expresses facts, but in a tone that expresses guidance for contributors to follow, since this is the purpose of this file.

Reply:
ACK. (I hope you were thinking something like this. :) )